sfge/src/main/java/com/salesforce/graph/symbols/apex/ApexIterableCollectionValue.java

```diff
@@ -0,0 +1,19 @@
+package com.salesforce.graph.symbols.apex;
+
+import com.salesforce.graph.vertex.Typeable;
+import java.util.Collection;
+import java.util.Optional;
+
+/** Represents ApexValues that hold an iterable collection of values. */
```

Comment:
New interface to represent List and Set. Other data structures in Apex can't be iterated.